Fixed a bug for fetch-incidents [Exabeam] (#23799)
demisto/content: Packs/Exabeam/ReleaseNotes/2_2_1.md

#### Integrations
##### Exabeam

> - Fixed an issue where ***fetch-incidents*** did not work when using with api key.

Reviewer: ```suggestion
- Fixed an issue where ***fetch-incidents*** failed when using with api key.
```

> - Fixed an issue where ***fetch-incidents*** does not work when using with api key.

Reviewer: ```suggestion
- Fixed an issue where ***fetch-incidents*** did not work when using with api key.
```
